src/main.rs

```diff
@@ -59,6 +65,12 @@ pub struct CliArgs {
     output_file: Option<String>,
 }
 
+#[derive(Serialize, Deserialize)]
+pub enum CoreContractMode {
+    Production,
+    Dev,
+}
+
 #[derive(Serialize, Deserialize)]
 pub struct ConfigFile {
```

Comment:
i don't know what's a good design to do this but looking at the config file, i don't know what optional things are needed for what commands. like `l1_eth_bridge_address` is needed for upgrades but not for others. we can add a TODO with this problem and solve it eventually